apps/client/src/pages/IntroPage.js

```diff
@@ -8,16 +8,20 @@ import Loading from "../components/Loading";
 import Nav from "../components/Nav";
 import { getDataNeedPageOrNext } from "../config/autofill";
 import withChecker from "../hoc/withChecker";
+import DefaultIntro from "../intros/shared/DefaultIntro";
```

Comment:
This can be changed to LocationIntro, right?

Reply:
Will check.